src/node_builtins.cc

```diff
@@ -647,6 +647,16 @@ void BuiltinLoader::HasCachedBuiltins(const FunctionCallbackInfo<Value>& args) {
       args.GetIsolate(), instance->code_cache_->has_code_cache));
 }
 
+void SetInternalLoaders(const FunctionCallbackInfo<Value>& args) {
+  Realm* realm = Realm::GetCurrent(args);
+  CHECK(args[0]->IsFunction());
+  CHECK(args[1]->IsFunction());
```

Comment:
Would it make sense(or make a difference) if this was `DCHECK`? Logically speaking, it does not look like the arguments would change according to debug/release builds.

Reply:
Not a lot, this is just following the convention we use for this sort of checks everywhere.